packages/webapp/src/store/api/types/index.ts

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
Just wondering if the redux store is the best location for types. As we start converting our codebase to .ts I get the feeling we will be using types in our containers or components or even backend too.

Reply:
Discussed in tech daily but want to leave a ping to discuss with @antsgar too.

Yeah I think your idea of having an animal type that we use throughout the frontend (and then extend to get the `apiSlice` interface) makes sense -- and I agree, must avoid interface duplication 👍 👍 👍 Would love to discuss types organization generally as even within the RTK Query docs alone there were several patterns.